airflow/utils/db.py

```diff
@@ -713,8 +713,8 @@ def initdb(session: Session = NEW_SESSION, load_connections: bool = True):
     if conf.getboolean("database", "LOAD_DEFAULT_CONNECTIONS") and load_connections:
         create_default_connections(session=session)
     # Add default pool & sync log_template
-    add_default_pool_if_not_exists()
-    synchronize_log_template()
+    add_default_pool_if_not_exists(session=session)
+    synchronize_log_template(session=session)
```

Comment:
Not strictly related, but it seems like a good idea to reuse the session here. `upgradedb` already does the same.